app/src/main/kotlin/com/simplemobiletools/notes/helpers/DBHelper.kt

```diff
@@ -138,25 +138,20 @@ class DBHelper private constructor(private val mContext: Context) : SQLiteOpenHe
         return note
     }
 
-    fun getNote(path: String): Note? {
-        val cols = arrayOf(COL_ID, COL_TITLE, COL_VALUE, COL_TYPE)
+    fun getNoteId(path: String): Int? {
+        val cols = arrayOf(COL_ID)
         val selection = "$COL_PATH = ?"
         val selectionArgs = arrayOf(path)
-        var note: Note? = null
         var cursor: Cursor? = null
         try {
             cursor = mDb.query(TABLE_NAME, cols, selection, selectionArgs, null, null, null)
             if (cursor?.moveToFirst() == true) {
-                val id = cursor.getIntValue(COL_ID)
-                val title = cursor.getStringValue(COL_TITLE)
-                val value = cursor.getStringValue(COL_VALUE)
-                val type = cursor.getIntValue(COL_TYPE)
-                note = Note(id, title, value, type, path)
+                return cursor.getIntValue(COL_ID)
             }
         } finally {
             cursor?.close()
         }
-        return note
+        return null
```

Comment:
Id like to avoid null integers, better use 0. Real notes start from 1.

Reply:
As you like, done.